lib/item_stream_handler.rb

```diff
@@ -79,14 +80,26 @@ def convert_record_to_checkout(decoded_records)
       .compact
   end
 
+  def dedup_checkout(checkout)
```

Comment:
Could this be `is_duplicate?` or something to indicate it returns a boolean that indicates whether or not the given checkout is a duplicate?

Reply:
Changed!